src/drivers/spi/Mybuild

```diff
@@ -1,6 +1,15 @@
 
 package embox.driver.spi
 
+module core {
+	option number log_level=0
+
+	@IncludeExport(path="drivers")
+	source "spi.h"
```

Comment:
Where are `spi.h` and `spi.c`? I do not see these file in the PR

Reply:
Forgot to add it, check last commits.